react/src/App.css

```diff
@@ -0,0 +1,102 @@
+.bookSearch {
+  max-width: 792px;
+  margin: 0 auto; }
```

Comment:
I'd put `}` in a newline so it keeps consistency with `.scss` files

Reply:
This file is being generated by sass.